### ISSUE TITLE
Roboto Mono is too widely spaced

### ISSUE THREAD

**Issue by tttom64** [termux/termux-styling#26]
The Roboto Mono font is too widely spaced, making it effectively unusable. It also seems to all be in bold type:
![roboto](https://user-images.githubusercontent.com/13117753/33067378-145cb218-cf02-11e7-9874-392e931a807c.jpg)

Iosevka, for comparison:
![iosevka](https://user-images.githubusercontent.com/13117753/33067392-20d1c02e-cf02-11e7-8968-3d811821c802.jpg)

This is what it should look like, taken from my desktop:
![ideal-spacing](https://user-images.githubusercontent.com/13117753/33067403-261d38a6-cf02-11e7-97c6-b50c4ff23ced.jpg)

**Comment by fornwall** [comment]
Hm, it looks like the Robot Mono fonts from nerd-tre (https://github.com/ryanoasis/nerd-fonts/raw/master/patched-fonts/RobotoMono/complete/Roboto%20Mono%20Nerd%20Font%20Complete.ttf) adds a lot of space compared to e.g. https://github.com/google/fonts/raw/master/apache/robotomono/RobotoMono-Regular.ttf.

I've asked a question at https://gitter.im/ryanoasis/nerd-fonts about it.

**Comment by tttom64** [comment]
That's strange. I installed `RobotoMono Nerd Font` on my desktop (using `gnome-terminal`) and the spacing seemed to be identical to `Roboto Mono`. So I expect it's not an issue with the font itself.

**Comment by tttom64** [comment]
Having downloaded both the font you linked and the one in this repo, there's a clear difference between the two.
From your link:
![roboto-nerd-font-regular](https://user-images.githubusercontent.com/13117753/33507438-d36688ca-d748-11e7-8d0e-b90c1e5d5d56.png)

From here:
![roboto-nerd-font-termux](https://user-images.githubusercontent.com/13117753/33507440-d3e14b28-d748-11e7-9313-1d7d5093cbe5.png)

**Comment by fornwall** [comment]
Indeed - I've created an issue at https://github.com/ryanoasis/nerd-fonts/issues/220 with your screenshots.

**Comment by tttom64** [comment]
I think you're mistaken - both of the fonts in the screenshots are from nerd-fonts, not google. The first one is from ryanoasis/nerd-fonts, and the second one is straight from termux/styling. So I think the issue is with the latter and not the former.

**Comment by fornwall** [comment]
Ah, sorry for being confused.

I actually wrote the wrong link to the nerd-tree font file used, it should be (note the `Mono` at the end):

- https://github.com/ryanoasis/nerd-fonts/raw/master/patched-fonts/RobotoMono/complete/Roboto%20Mono%20Nerd%20Font%20Complete%20Mono.ttf

Does this font have the spacing issue, while

- https://github.com/google/fonts/raw/master/apache/robotomono/RobotoMono-Regular.ttf

does not?

**Comment by tttom64** [comment]
Yes. Neither `RobotoMono-Regular.ttf` nor `Roboto Mono Nerd Font Complete.ttf` have the issue, but `Roboto Mono Nerd Font Complete Mono.ttf` does.

**Comment by tttom64** [comment]
I understand that you're waiting on a fix from ryanoasis/nerd-fonts, but as a temporary fix, I suggest removing the final 'Mono' from the end of the filename in the .sh setup script. Would this work?
